tools/ansible/tasks/logging/install_fluentd.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Create temporary directory on windows host
+  win_tempfile:
+    state: directory
+  register: tempdir_win
+- name: Fetch ruby devkit exe         # Ruby version < 2.6 and >= 2.0, 64 bit
+  win_get_url:
+    url: "{{ ruby_fetch_url }}"
+    dest: "{{ tempdir_win.path }}\\"
+  register: ruby_devkit
+- name: Check install path
+  win_stat:
+    path: "{{ install_dir }}"
+  when: install_dir is defined
+  register: install_dir_exists
+- name: Install ruby to particular directory
+  win_shell: "{{ ruby_devkit.dest }} /VERYSILENT /SUPPRESSMSGBOXES /DIR={{ install_dir }}"
+  when: install_dir is defined and install_dir_exists.stat.exists
+- name: Install ruby
+  win_shell: "{{ ruby_devkit.dest }} /VERYSILENT /SUPPRESSMSGBOXES"
+  when: install_dir is not defined or
+        (install_dir_exists is defined and install_dir_exists.stat.exists==false)
+- name: Check ruby installed
+  win_shell: "ruby --version"
```

Comment:
This assumes that `ruby` is in the executable path and I am guessing the ruby devkit does that for you. So please  call that assumption out in a comment here. Same goes for `gem`.

Reply:
It is not obvious to the reader without having prior knowledge

Reply:
I see what you mean. Okay.
In that case we should also call out why ruby-devkit instead of simple ruby install. Ruby devkit is a bundle of Ruby as well as MSYS2 packages, and both are required to install fluentd gem